src/components/PokemonDetails/PokemonDetails.js

```diff
@@ -0,0 +1,88 @@
+import React, { useState, useEffect } from "react";
+import "./PokemonDetails.css";
+
+const PokemonDetails = ({ foundPokemon, getPokemonImage }) => {
+  const [pokemonDetails, setPokemonDetails] = useState([]);
+  const [error, setError] = useState("");
+
+  const pokemonImage = getPokemonImage(
+    foundPokemon[0].url.replace(/\D/g, "").slice(1)
+  );
+
+  const getPokemonDetails = async () => {
+
+    let pokemonId = foundPokemon[0].url.replace(/\D/g, "").slice(1);
+    const url = `https://pokeapi.co/api/v2/pokemon/${pokemonId}`;
+
+    try {
+      const res = await fetch(url);
+      const pokemonDetails = await res.json();
+      setPokemonDetails(pokemonDetails);
+    } catch (error) {
+      setError(error.message);
+    }
+  };
+// for some reason I don't know yet --it is not reseting the state when we type a diferent name or id 
+// i'm not sure if only clearing the search it is what we need, probably we need a way to look into the details again..
+  useEffect(() => {
+    getPokemonDetails();
+  }, []);
```

Comment:
I'm still unclear on all of the nuances of Hooks, if we change this to a class based with component did mount, I can be more helpful. 
But...
If we _first_, clear out the state of foundPokemon in app from this useEffect, and _then_ make the fetch, that could solve this issue.